models/models_methods.go

```diff
@@ -1126,9 +1126,6 @@ func CreateStepIDDataFromString(compositeVersionStr, defaultStepLibSource string
 	}
 
 	version := getStepVersion(compositeVersionStr)
-	if src == "git" && version == "" {
-		version = "master"
```

Comment:
Curious if we know how to quickly roll this back if this assumption is required for the system?

Reply:
I wonder this as well. I'm waiting to merge this until @godrei is back to get his insight since he knows this codebase the best.

Reply:
or better question does git just know what to do here?

Reply:
FYI it's not publicly documented that you can use the `git::` scheme without specifying the branch part: https://devcenter.bitrise.io/en/references/steps-reference/step-reference-id-format.html#the-git---source

So I wouldn't worry about it too much.

Reply:
ah nice. Well maybe we should update the docs too then once this update lands. :)